plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java

```diff
@@ -93,6 +93,13 @@
 
 	boolean isRoot();
 
+	/**
+	 * Calculates the depth of the subtree starting from this feature. If this feature has no children, the depth is 0.
+	 * 
+	 * @return The depth of the subtree starting from this feature
+	 */
+	int getSubtreeDepth();
```

Comment:
@h3ssto Is the change to the interface required? It is only used one and seems to be more like an implementation for a FeatureStructureUtils or directly where it is used. When we change the interface then all implementations need to be adjusted.

Reply:
Yes, I also think this is unnecessary.